setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(name='castor',
+      version='1.0.0',
+      description='models for question answering',
+      packages=['sm_model'],
+      )
```

Comment:
http://stackoverflow.com/questions/6609956/python-function-argument-list-formatting
i got confused by this but it's not a big deal. the comma at the end makes no sense though. ship it.